projects/dsp-ui/src/lib/viewer/representation/upload-form/upload-form.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { NotificationService } from '../../../action/services/notification.service';
+import { UploadedFileResponse, UploadFileService } from '../../services/upload-file.service';
+
+@Component({
+    selector: 'dsp-upload-form',
+    templateUrl: './upload-form.component.html',
+    styleUrls: ['./upload-form.component.scss']
+})
+export class UploadFormComponent implements OnInit {
+
+    @Input() readonly resourceTyoe = 'Image'; // only StillImageRepresentation supported so far
+    @Output() file: File;
```

Comment:
Wouldn't an `@Output` require an `EvenEmitter`?

Reply:
> Wouldn't an `@Output` require an `EvenEmitter`?

As we spoke yesterday, `@Output` was removed with fef4c59 due to not know specification of usage.

Reply:
ok